docker-compose-flask-microservices.yml

```diff
@@ -0,0 +1,8 @@
+version: '3.8'
+
+services:

```

Comment:
We should define also a reverse-proxy.
you can check how we do it here:
[https://github.com/rogowski-piotr/system-benchmarking-monolith-vs-microservices/blob/main/docker-compose-spring-boot-microservices.yml](https://github.com/rogowski-piotr/system-benchmarking-monolith-vs-microservices/blob/main/docker-compose-spring-boot-microservices.yml)